src/components/HelperApp/HelperApp.js

```diff
@@ -8,7 +8,7 @@ class HelperApp extends React.Component {
   static propTypes = {
     match: PropTypes.object,
     stripes: PropTypes.object,
-    close: PropTypes.func,
+    onClose: PropTypes.func,
```

Comment:
As far as I understand, It was changed from close to onClose in one place. Could you please clarify if it was broken before or this change will affect the behavior?